go.sum

```diff
@@ -375,4 +375,4 @@ k8s.io/kube-openapi v0.0.0-20190228160746-b3a7cee44a30/go.mod h1:BXM9ceUBTj2QnfH
 k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a h1:2jUDc9gJja832Ftp+QbDV0tVhQHMISFn01els+2ZAcw=
 k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
 sigs.k8s.io/yaml v1.1.0 h1:4A07+ZFc2wgJwo8YNlQpr1rVlgUDlxXHhPJciaPY5gs=
-sigs.k8s.io/yaml v1.1.0/go.mod h1:UJmg0vDUVViEyp3mgSv9WPwZCDxu4rQW1olrI1uml+o=
+sigs.k8s.io/yaml v1.1.0/go.mod h1:UJmg0vDUVViEyp3mgSv9WPwZCDxu4rQW1olrI1uml+o=
```

Comment:
This should not be a change? Is there maybe a newline or something?

Reply:
I copied the code from master, to verify and it's the same.
Not really sure, what's going on.

Reply:
It seems that a newline at the end of the file was removed somehow. I had a similar problem at some point, not sure how I fixed it.

Reply:
Try `git checkout origin/master -- go.sum; git add go.sum`. (Don't use `add -p` as it won't recognise it as a change).

Reply:
Fixed with this